.github/workflows/npm-brew.yaml

```diff
@@ -45,10 +45,8 @@ jobs:
           NPM_ENVIRONMENT: "production"
 
   homebrew:
-    # The newest version of code-server needs to be available on npm when this runs
-    # otherwise, it will 404 and won't open a PR to bump version on homebrew/homebrew-core
     needs: npm
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
🛠️ [Homebrew runs on Linux](https://github.com/Homebrew/actions/tree/master/setup-homebrew#setup-homebrew-github-action) and Linux runners are more available (anecdotally) than macOS runners. Maybe it'll save some time.